workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/Commands/Create.cs

```diff
@@ -0,0 +1,191 @@
+                    RegionCode = options.Region
+                };
+
+                if (options.SnapshotPath != null)
+                {
+                    var snapshotId = UploadSnapshot(snapshotServiceClient, options.SnapshotPath, options.ProjectName,
+                        options.DeploymentName);
+
+                    if (string.IsNullOrEmpty(snapshotId))
+                    {
+                        return 1;
+                    }
+
+                    deployment.StartingSnapshotId = snapshotId;
+                }
+
+                if (options.Tags != null)
+                {
+                    foreach (var tag in options.Tags)
+                    {
+                        deployment.Tag.Add(tag);
+                    }
+                }
+
+                var deploymentOp = deploymentServiceClient.CreateDeployment(new CreateDeploymentRequest
+                {
+                    Deployment = deployment
+                }).PollUntilCompleted();
+
+                if (deploymentOp.Result.Status != Deployment.Types.Status.Running)
+                {
+                    Ipc.WriteError(Ipc.ErrorCode.Other, "Deployment failed to start for an unknown reason.");
+                    return 1;
+                }
+            }
+            catch (Grpc.Core.RpcException e)
+            {
+                if (e.Status.StatusCode == Grpc.Core.StatusCode.NotFound)
+                {
+                    Ipc.WriteError(Ipc.ErrorCode.NotFound, e.Status.Detail);
+                    return 1;
+                }
+
+                throw;
+            }
+
+            return 0;
+        }
+
+        private static string ModifySimulatedPlayerLaunchJson(Options.CreateSimulated options)
+        {
+            var playerAuthServiceClient = PlayerAuthServiceClient.Create();
+
+            // Create development authentication token used by the simulated players.
+            var dat = playerAuthServiceClient.CreateDevelopmentAuthenticationToken(
+                new CreateDevelopmentAuthenticationTokenRequest
+                {
+                    Description = "DAT for sim worker deployment.",
+                    Lifetime = Duration.FromTimeSpan(new TimeSpan(7, 0, 0, 0)),
+                    ProjectName = options.ProjectName
+                });
+
+            // Add worker flags to sim deployment JSON.
+            var devAuthTokenIdFlag = new JObject
+            {
+                { "name", $"{options.FlagPrefix}_simulated_players_dev_auth_token_id" },
+                { "value", dat.DevelopmentAuthenticationToken.Id }
+            };
+
+            var targetDeploymentFlag = new JObject
+            {
+                { "name", $"{options.FlagPrefix}_simulated_players_target_deployment" },
+                { "value", options.TargetDeployment }
+            };
+
+            var launchConfigRaw = File.ReadAllText(options.LaunchJsonPath);
+            dynamic launchConfig = JObject.Parse(launchConfigRaw);
```

Comment:
👀

Reply:
The alternative is to define the JSON layout of the launch configuration 😬

Reply:
It's probably fine, just worth documenting, I think